_maps/skyrat/automapper/automapper_config.toml

```
@@ -100,14 +100,6 @@ required_map = "DeltaStation2.dmm"
 coordinates = [121, 124, 1]
 trait_name = "Station"
```

```suggestion
trait_name = "foobar"
```
checks are fucked and I want it to rerun